### PR TITLE
Handle "403" error (provider)

### PR DESCRIPTION
**Pull request by Fadiabb**
* #94

### CODE REVIEW

File: cmd/csaf_uploader/main.go [hunk 1]
```diff
@@ -296,6 +296,9 @@ func (p *processor) process(filename string) error {
 	if resp.StatusCode != http.StatusOK {
 		uploadErr = fmt.Errorf("upload failed: %s", resp.Status)
 		fmt.Printf("HTTPS %s\n", uploadErr)
+		if resp.StatusCode == http.StatusForbidden {
+			return err
```
Comment (bernhardreiter) [inline cmd/csaf_uploader/main.go:300]: @s-l-teichmann 
We can upload several files, should be try them all, even if we are getting errors on some or abort on the first error?

Comment (s-l-teichmann) [reply cmd/csaf_uploader/main.go:300]: @bernhardreiter Yes, maybe we should continue with the next one.

Comment (bernhardreiter) [reply cmd/csaf_uploader/main.go:300]: @s-l-teichmann right now, we do. Okay, then this remark is resolved.

File: cmd/csaf_uploader/main.go [hunk 2]
```diff
@@ -296,6 +296,9 @@ func (p *processor) process(filename string) error {
 	if resp.StatusCode != http.StatusOK {
 		uploadErr = fmt.Errorf("upload failed: %s", resp.Status)
 		fmt.Printf("HTTPS %s\n", uploadErr)
+		if resp.StatusCode == http.StatusForbidden {
```
[the remark this answers is not in the document]
Comment (bernhardreiter) [reply cmd/csaf_uploader/main.go:299]: And we need to print out which file failed, if we did not get this back from the server.

**Comment by bernhardreiter** [comment]
We plan to solve this with a different pull request.

**Pull request closed without merging**